src/index.html

```diff
@@ -4,9 +4,216 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" value="crossorigin">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" value="crossorigin">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" value="crossorigin">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div>
+      <header class="head">
+        <a><img src="./images/logo.png" alt="logo" class="Logo"></a>
+
+        <nav class="Nav">
+          <ul class="Nav__ALL">
+            <li class="Nav__list">
+              <a href="#" class="Nav__1 is-active">Apple</a>
+            </li>
+            <li class="Nav__list">
+              <a class="Nav__2">Samsung</a>
+            </li>
+            <li class="Nav__list">
+              <a class="Nav__3">СМАРTФОНЫ И ТЕЛЕФОНЫ</a>
+            </li>
+            <li class="Nav__list">
+              <a class="Nav__4" data-qa="nav-hover">Ноутбуки и компьютеры</a>
+            </li>
+            <li class="Nav__list">
+              <a class="Nav__5">Гаджеты</a>
+            </li>
+            <li class="Nav__list">
+              <a class="Nav__6">Планшеты</a>
+            </li>
+            <li class="Nav__list">
+              <a class="Nav__7">ФОТО</a>
+            </li>
+            <li class="Nav__list">
+              <a class="Nav__8">ВИДЕО</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <main class="Block">
+        <div class="All_container">
+          <div class="Card" data-qa="card">
```

Comment:
we write classes with lowercase first letter
```suggestion
          <div class="card" data-qa="card">
```